### PR TITLE
ipsumdump 1.84 (new formula)

### PR DESCRIPTION
**Pull request by mavam**
The ipsumdump program summarizes TCP/IP dump files into a self-describing ASCII format easily readable by humans and programs. Here's a few lines from an example summary:

```
!IPSummaryDump 1.3
!creator "ipsumdump -sd /home/kohler/largedump.gz"
!host max.lcdf.org
!runtime 1092201225.541624 (Tue Aug 10 22:13:45 2004)
!data ip_src ip_dst
64.55.139.202 209.247.204.242
18.26.4.9 64.55.139.202
```

### CODE REVIEW

File: Library/Formula/ipsumdump.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  head "https://github.com/kohler/ipsumdump.git"
+
+  def install
+    system "./configure", "--prefix#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/ipsumdump", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/ipsumdump.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (mavam) [inline Library/Formula/ipsumdump.rb]: Fixed. Hope this works now!

File: Library/Formula/ipsumdump.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  head "https://github.com/kohler/ipsumdump.git"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"one-byte-payload.pcap").write(
```
Comment (mikemcquaid) [inline Library/Formula/ipsumdump.rb]: Please see if `test_fixtures("test.pcap")` is suitable, thanks!

Comment (mavam) [inline Library/Formula/ipsumdump.rb]: That'll certainly do. Good to know that such a fixture exists.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**